src/main/java/kr/allcll/seatfinder/exception/ExceptionMessage.java

```diff
@@ -0,0 +1,19 @@
+package kr.allcll.seatfinder.exception;
+
+public enum ExceptionMessage {
+
+    MAX_PIN_EXCEPTION("이미 %d개의 핀을 등록했습니다."),
+    EXIST_PIN_EXCEPTION("이미 핀 등록된 과목입니다."),
+    PIN_AND_SUBJECT_NOT_MATCH("핀에 등록된 과목이 아닙니다."),
+    NOT_EXIST_SUBJECT("존재하지 않는 과목 입니다.");
```

Comment:
에러 코드가 좀더 일관성 있고 직관적으로 작성하면 좋을 거 같아요~ 
아래는 제 제안인데, 이건 코드 리뷰하면서 맞춰나가보아요! 

```suggestion
    PIN_LIMIT_EXCEEDED("핀할 수 있는 과목 수를 초과했습니다. 최대 {limit}개의 과목만 핀할 수 있습니다"),
    DUPLICATE_PIN("{subject}은 이미 핀 등록된 과목입니다."),
    PIN_SUBJECT_MISMATCH("핀에 등록된 과목이 아닙니다."),
    SUBJECT_NOT_FOUND("존재하지 않는 과목 입니다. 입력한 과목명: {subject}");
```

Reply:
반영한 것 확인해주세요

> SUBJECT_NOT_FOUND("존재하지 않는 과목 입니다. 입력한 과목명: {subject}");

이건 사용자에게 입력 받는 로직에서 실행되는게 아니라서 그대로 남겼습니당